cmd/kubeadm/app/cmd/init.go

```diff
@@ -184,6 +184,10 @@ func AddInitConfigFlags(flagSet *flag.FlagSet, cfg *kubeadmapiext.MasterConfigur
 		&cfg.NodeName, "node-name", cfg.NodeName,
 		`Specify the node name.`,
 	)
+	flagSet.StringVar(
+		&cfg.ClusterName, "cluster-name", cfg.ClusterName,
```

Comment:
Set `ClusterName` to `"kubernetes"` by default in L110.

```
cfg := &kubeadmapiext.MasterConfiguration{ClusterName: "kubernetes"}
```

Reply:
Good call.

Reply:
So we have a moratorium on flags, I'd be ok with a config option with defaults. 

Also, if it's part of the struct I sure hope we don't need to change so many signatures.